### PR TITLE
Add helpful debug logs in PP and minor cleanup

### PR DESCRIPTION
**Pull request by jiivan**
In develop `TransactionSystem.get_funds_needed` will be exposed via rpc

### CODE REVIEW

**Review by Krigpl — COMMENTED** [review]
What is this needed for?

**Comment by jiivan** [comment]
@Krigpl for a requestor to know how much funds he needs to pay his debts

File: golem/ethereum/transactionsystem.py [hunk 1]
```diff
@@ -345,6 +344,11 @@ def get_locked_gnt(self) -> int:
             raise Exception('Start was not called')
         return self._gntb_locked + self._payment_processor.reserved_gntb
 
+    def get_funds_needed(self) -> dict:
```
Comment (Krigpl) [inline golem/ethereum/transactionsystem.py]: If you introduced a named tuple then just stick to it

**Review by Krigpl — COMMENTED** [review]
> In develop TransactionSystem.get_funds_needed will be exposed via rpc

I don't get something - it's gonna be exposed in develop, but the PR is to b0.18. How is it gonna be used in this branch and why it's not gonna be exposed via rpc here?

File: golem/ethereum/transactionsystem.py (hunk 1 above)
Comment (jiivan) [reply golem/ethereum/transactionsystem.py]: NamedTuple won't go through RPC. It's only reason for this pass through method to make results RPC ready.

File: golem/ethereum/paymentprocessor.py [hunk 2]
```diff
@@ -141,14 +141,31 @@ def __get_next_batch(self, closure_time: int) -> int:
                 break
             gntb_balance -= p.value
             if gntb_balance < 0:
+                log.debug(
+                    'Insufficient GNTB balance.'
+                    ' value=%(value).6f, subtask_id=%(subtask)s',
+                    {
+                        'value': p.value,
```
Comment (Krigpl) [inline golem/ethereum/paymentprocessor.py]: `/ denoms`

**Review by Krigpl — APPROVED** [review]

**Pull request merged**